planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/util/DefaultConstants.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2014 SeaClouds
+ * Contact: SeaClouds
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package eu.seaclouds.platform.planner.optimizer.util;
+
+public class DefaultConstants {
+
+   public static final String DEFAULT_BENCHMARK_INFORMATION_PATH    = "./src/main/resources/benchmark-platforms.yml";
```

Comment:
Probably, it may be better to find this file as a resource instead of use the path